src/tallies/filter.cpp

```diff
@@ -19,6 +19,7 @@
 #include "openmc/tallies/filter_distribcell.h"
 #include "openmc/tallies/filter_energyfunc.h"
 #include "openmc/tallies/filter_energy.h"
+#include "openmc/tallies/filter_collision.h"
```

Comment:
Put this before distribcell so it's all in alphabetical order please :slightly_smiling_face:

Reply:
1. I can't believe I did not notice that this was in alphabetical order
2. I'm so happy this is something you pick up on.

I'll change it :)